sdk/testing/src/main/java/io/opentelemetry/sdk/testing/exporter/InMemoryMetricReaderBuilder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.testing.exporter;
+
+import static io.opentelemetry.sdk.metrics.export.MemoryMode.IMMUTABLE_DATA;
+
+import io.opentelemetry.sdk.metrics.export.AggregationTemporalitySelector;
+import io.opentelemetry.sdk.metrics.export.DefaultAggregationSelector;
+import io.opentelemetry.sdk.metrics.export.MemoryMode;
+
+public final class InMemoryMetricReaderBuilder {
+  private AggregationTemporalitySelector aggregationTemporalitySelector =
+      AggregationTemporalitySelector.alwaysCumulative();
+  private DefaultAggregationSelector defaultAggregationSelector =
+      DefaultAggregationSelector.getDefault();
+  private MemoryMode memoryMode = IMMUTABLE_DATA;
+
+  /**
+   * Creates an {@link InMemoryMetricReaderBuilder} with defaults.
+   *
+   * <p>Creates a builder with always-cumulative {@link AggregationTemporalitySelector}, default
+   * {@link DefaultAggregationSelector} and {@link MemoryMode#IMMUTABLE_DATA} {@link MemoryMode}
+   */
+  InMemoryMetricReaderBuilder() {}
+
+  public InMemoryMetricReaderBuilder setAggregationTemporalitySelector(
+      AggregationTemporalitySelector aggregationTemporalitySelector) {
+    this.aggregationTemporalitySelector = aggregationTemporalitySelector;
+    return this;
+  }
+
+  /**
+   * Sets the {@link DefaultAggregationSelector}
```

Comment:
Build is failing because this summary doc is missing a period: https://github.com/open-telemetry/opentelemetry-java/actions/runs/6095176672/job/16538202714?pr=5709#step:5:1140

Reply:
Added. Thanks!